src/blockdata/transaction.rs

```diff
@@ -220,12 +221,204 @@ impl Default for TxIn {
         TxIn {
             previous_output: OutPoint::default(),
             script_sig: Script::new(),
-            sequence: u32::max_value(),
+            sequence: Sequence::MAX,
             witness: Witness::default(),
         }
     }
 }
 
+/// Bitcoin transaction input sequencing number.
```

Comment:
I don't remember ever hearing "sequencing number", "sequence number" is probably the correct word.